hardhat.config.ts

```diff
@@ -35,12 +36,20 @@ const config: HardhatUserConfig = {
     currency: "USD",
   },
   etherscan: {
-    apiKey: process.env.ETHERSCAN_API_KEY,
+    apiKey: {
+      ropsten: process.env.ETHERSCAN_API_KEY || "",
```

Comment:
should these be `ETHERSCAN_ROPSTEN_API_KEY` etc?  Also maybe add em to .env.template if we got one?